dsrc/parser.lit

```diff
@@ -484,6 +489,8 @@ else if (matchAll(line, regex("^---.+"))) {
     curBlock.isCodeblock = true;
     curBlock.name = strip(line[3..$]);
 
```

Comment:
Inside the check for  `nameb` from the regex (which will contain any of the old uses of `+=`) I will check if it ends with a `+=` or `:=` this way going forward ether syntax will work. (`nameb` only returns the name of the block if there is no second `---` present)

Otherwise this would break on something like `--- Block Name --- += noWeave`